pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.js

```diff
@@ -59,34 +60,27 @@ router.post('/', function(req, res, next) {
     }
 });
 
-router.get('/', function(req, res, next) {
+router.get('/', asyncHandler(async (req, res, next) => {
```

Comment:
An intermediate implementation used an async `renderHtmlOrText`, so I converted this to an async handler. Though it's not necessary any more, I'm leaving it as this is paying down a bit of tech debt.